packages/experimental/Expander/src/Expander.tsx

```diff
@@ -22,20 +22,29 @@ export const Expander = compose<ExpanderType>({
     const Root = useSlots(userProps).root;
     const [expandedState, setExpandedState] = React.useState(userProps.expanded);
     const expanderHeight = expandedState? userProps.expandedHeight : userProps.collapsedHeight;
-    async function onExpanderChange(event: ExpanderChangeEvent) {
-      if (event.nativeEvent.expanded != null) {
-        event.persist();
-        if (!event.nativeEvent.expanded) {
-          await delay(175);
-        }
-        setExpandedState(event.nativeEvent.expanded);
+
+    const _onCollapsing = async () => {
+      if (userProps.onCollapsing !== undefined) {
+        userProps.onCollapsing();
       }
+      // Need to delay the height change so that the animation runs
+      await delay(175);
```

Comment:
nit - I wonder if there is a better way to be notified of "the animation has completed", e.g. [Completed](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.animation.timeline.completed?view=winrt-20348) event.

Reply:
What's the feature we're enabling w/ an animation completed callback?

Reply:
When the Expander is collapsed or expanded, we manually adjust the height of the Expander. For the collapsing event, the height is changed too early and clips the native control's collapsing animation, so we add a slight delay to the thread before making the height adjustment.